src/assets/stylesheets/index.scss

```diff
@@ -859,9 +859,9 @@ section .card.no-color-adaption {
 
 img.img-flag {
   display: inline-block;
-  height: 15px;
   margin: -3px 6px 0 0;
-  max-width: 17px;
+  max-height: 15px;
```

Comment:
In edit mode (with `1.49.0-rc1`) there is still a flicker visible:
<img width="673" alt="Screenshot 2025-04-01 at 08 26 26" src="https://github.com/user-attachments/assets/5b9c0dac-5814-44ad-9fee-4f6c000253e6" />

Full video:

https://github.com/user-attachments/assets/f673b867-1b38-4108-bb99-7cc09af5ba40